docs/examples/cookbook/animated-computed.md

```diff
@@ -87,7 +87,7 @@ independently.
 
 ```Lua
 		Position = scope:Tween(
-			scope:Computed(function(use)
+			scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used

Reply:
```suggestion
			scope:Computed(function(use)
```

Reply:
reverted